docs/sdk/serverless-signing/combining-signatures.md

```diff
@@ -0,0 +1,86 @@
+import FeedbackComponent from "@site/src/pages/feedback.md";
+
+# Combine Signatures Within an Action
+
+## Overview 
+
+Within a lit action, you may choose to combine a signature from a `pkp` from within the context of an action. Meaning signature shares from each node will be combined and given to every node which recieved a request to execute the given action. Combining within an action may be useful if you wish to take advantage of the `secure compute enviorment` offered by the Lit network. Actions which take combine signatures from within their own execution context will no provide the shares of the signatures to the client. So all information will stay in the trusted execution enviorment (TEE).
```

Comment:
Copy suggestion: 

Current: 
"Within a lit action, you may choose to combine a signature from a `pkp` from within the context of an action. Meaning signature shares from each node will be combined and given to every node which recieved a request to execute the given action. Combining within an action may be useful if you wish to take advantage of the `secure compute enviorment` offered by the Lit network. Actions which take combine signatures from within their own execution context will no provide the shares of the signatures to the client. So all information will stay in the trusted execution enviorment (TEE)."

Suggestion:
"When you [sign a message](https://developer.litprotocol.com/v3/sdk/serverless-signing/quick-start) with Lit, `signature shares` are typically combined client-side to form the complete signature. The `signAndCombineEcdsa` function allows you to combine these shares within a Lit Action, meaning they will remain within the confines of each [Lit node's TEE](https://developer.litprotocol.com/v3/resources/how-it-works#1-lit-nodes) and not exposed to the end client. 

When you call `signAndCombineEcdsa`, the `signature shares` are collected from each Lit node and combined on a _single_ node."

^^ that function is only run on a single node right?

Reply:
@a1ttech  - like decryption, sign & combine is performed on all nodes.    The resulting signature should probably only be used by a single node, at which time you would typically execute a `runOnce` command.